### PR TITLE
Add task to auto-generate Windows 2019 stemcell release notes

### PR DESCRIPTION
**Pull request by aramprice**
This task cribs from the existing task which generates the contents of the linux stemcell release notes.

The task which generates a _slightly_ different version of [windows-stemcell-v2019x.html.md.erb](https://github.com/pivotal-cf/docs-stemcell-rn/blob/master/windows-stemcell-v2019x.html.md.erb) than is currently present in the repo. From a quick review the changes appear to be _mostly_ cosmetic, or additive. 

NOTE: Information for some very old releases (circa 2019 / 2020) are missing or different because of the switch from `bosh-windows-stemcell-builder` => `stembuild`. We believe this is probably OK, though perhaps a manual "addendum" page to the auto-generated release notes would be appropriate.

If this PR is accepted then whichever CI executes the existing `ci/task.yml` will need to be configured to _also_ execute the `ci/task-windows.yml`. 

To run the generation code check out this branch, go into the repo and do the following:
```shell
# go into the ci/ directory
cd ci/

# install the required gems - the code expects ruby 3.2, though most likely any 3.x version of ruby will work
bundle install

# generate the release notes and place them into a file prefixed with `NEW-`
ruby get_windows_stemcells.rb > ../NEW-windows-stemcell-v2019x.html.md.erb
```

The result should be a `NEW-windows-stemcell-v2019x.html.md.erb` alongside `windows-stemcell-v2019x.html.md.erb` at the top level of this repository.

### CODE REVIEW

**Comment by aramprice** [comment]
[auto-generated_windows-stemcell-v2019x.html.md.erb](https://github.com/user-attachments/files/15809049/auto-generated_windows-stemcell-v2019x.html.md)

File: .DS_Store
Comment (selzoc) [inline .DS_Store:1]: Can we get this added to the `.gitignore`?

Comment (aramprice) [reply .DS_Store:1]: sure, that was a previous commit turd. I'll move the `.gitignore` up a level and add this

Comment (aramprice) [reply .DS_Store:1]: done

**Review by selzoc — APPROVED** [review]

**Pull request merged**